Upstream: Mailed
Source: Daniel Mierswa <impulze@impulze.org>
Reason: EOF needs <cstdio>

diff -Naur boolstuff-0.1.12.orig/src/commands/booldnf.cpp boolstuff-0.1.12/src/commands/booldnf.cpp
--- boolstuff-0.1.12.orig/src/commands/booldnf.cpp	2008-10-08 05:52:15.000000000 +0200
+++ boolstuff-0.1.12/src/commands/booldnf.cpp	2009-09-01 02:45:29.000000000 +0200
@@ -32,6 +32,7 @@
 #include <getopt.h>
 #endif
 
+#include <cstdio>
 #include <iostream>
 
 using namespace std;
